drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -793,6 +794,78 @@ static int bnxt_rss_hash_update_op(struct rte_eth_dev *eth_dev,
 	return 0;
 }
 
+int rte_pmd_bnxt_set_tx_loopback(uint8_t port, uint8_t on)
```

Comment:
It appears that this function will either do nothing, or return an error.  I don't see where this actually sets tx loopback.